hermes_core/tests/test_timedata.py

```diff
@@ -102,11 +102,52 @@ def test_timedata_default():
 
 
 def test_multidimensional_data():
+    # fmt: off
+    input_attrs = {
+        "Descriptor": "EEA>Electron Electrostatic Analyzer",
+        "Data_level": "l1>Level 1",
+        "Data_version": "v0.0.1",
+    }
+    # fmt: on
     ts = get_test_timeseries()
     ts["var"] = Quantity(value=random(size=(10, 2)), unit="s", dtype=np.uint16)
 
-    with pytest.raises(ValueError):
-        _ = TimeData(ts)
+    test_data = TimeData(ts, meta=input_attrs)
+
+    assert test_data["var"].shape == (10, 2)
+
+
+def test_suport_nrv_data():
+    # fmt: off
+    input_attrs = {
+        "Descriptor": "EEA>Electron Electrostatic Analyzer",
+        "Data_level": "l1>Level 1",
+        "Data_version": "v0.0.1",
+    }
+    # fmt: on
+    ts = get_test_timeseries()
+
+    # Bad Support
+    support_data = {"support_var": [1, 2, 3]}
+    with pytest.raises(TypeError):
+        _ = TimeData(ts, support_data=support_data, meta=input_attrs)
+
+    # Bad NRV
+    nrv_data = {"nrv_var": [1, 2, 3]}
+    with pytest.raises(TypeError):
+        _ = TimeData(ts, nrv_data=nrv_data, meta=input_attrs)
+
+    # Good Support and NRV
+    support_data = {"support_var": Column(data=[1, 2, 3])}
+    nrv_data = {"nrv_var": Column(data=[1, 2, 3])}
+
+    # Create TimeData
+    test_data = TimeData(
+        ts, support_data=support_data, nrv_data=nrv_data, meta=input_attrs
+    )
+
+    assert "support_var" in test_data
```

Comment:
I would also add a test to check that the data that you added is actually in the object, meaning the actual values.